{"uuid": "c39dcd58-dbbe-48cc-b799-2109a8ead407", "children": ["3bb4c0d5-a448-43fb-bed3-4d7f5aac5eb8"], "befores": [{"name": "setup", "status": "passed", "start": 1752584730996, "stop": 1752584735328}], "afters": [{"name": "setup::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Class' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\huyhe\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\huyhe\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 938, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Users\\huyhe\\OneDrive\\Desktop\\Selenium\\tests\\base_test.py\", line 24, in setup\n    if request.node.rep_call.failed:\n       ^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1752584749253, "stop": 1752584749273}], "start": 1752584730996, "stop": 1752584749273}